swmmio/tests/data/SWMMExtCntrlModel.inp

```diff
@@ -0,0 +1,2601 @@
+C16              CIRCULAR     1.5              0          0          0          1
+C17              CIRCULAR     1.5              0          0          0          1
+C18              CIRCULAR     1.5              0          0          0          1
+C19              CIRCULAR     1.5              0          0          0          1
+C2               CIRCULAR     1                0          0          0          1
+C20              CIRCULAR     1.5              0          0          0          1
+C21              CIRCULAR     1.5              0          0          0          1
+C22              CIRCULAR     1.5              0          0          0          1
+C23              CIRCULAR     1.5              0          0          0          1
+C24              CIRCULAR     1.5              0          0          0          1
+C25              CIRCULAR     1.5              0          0          0          1
+C26              CIRCULAR     1.5              0          0          0          1
+C27              CIRCULAR     1.5              0          0          0          1
+C28              CIRCULAR     1.5              0          0          0          1
+C29              CIRCULAR     1.5              0          0          0          1
+C3               CIRCULAR     1.5              0          0          0          1
+C30              CIRCULAR     1.5              0          0          0          1
+C4               CIRCULAR     1.5              0          0          0          1
+C5               CIRCULAR     1.5              0          0          0          1
+C6               CIRCULAR     2                0          0          0          1
+C7               CIRCULAR     2                0          0          0          1
+C8               CIRCULAR     3                0          0          0          1
+C9               CIRCULAR     3                0          0          0          1
+OR1              CIRCULAR     1.5              0          0          0
+OR2              CIRCULAR     1.5              0          0          0
+OR3              CIRCULAR     1.5              0          0          0
+
+[LOSSES]
+;;Link           Inlet      Outlet     Average    Flap Gate  SeepageRate
+;;-------------- ---------- ---------- ---------- ---------- ----------
+
+[INFLOWS]
+;;                                                 Param    Units    Scale    Baseline Baseline
+;;Node           Parameter        Time Series      Type     Factor   Factor   Value    Pattern
+;;-------------- ---------------- ---------------- -------- -------- -------- -------- --------
+J13              FLOW             10002_TimeSeries FLOW     1.0      1        0
+J18              FLOW             10002_TimeSeries FLOW     1.0      1        0
+J28              FLOW             10002_TimeSeries FLOW     1.0      1        0
+
+[TIMESERIES]
+;;Name           Date       Time       Value
+;;-------------- ---------- ---------- ----------
+10002_TimeSeries 10/01/2016 00:00      0.000356481487475
```

Comment:
@bemcdonnell do we need this timeseries in the inp for the unit tests? if not, can we drop this, just to keep the file size down a bit?

Reply:
@aerispaha probably not hugely required.  I could remove if you would like

Reply:
@bemcdonnell awesome, thank you!